bundles/io.openliberty.tools.eclipse.ui/src/io/openliberty/tools/eclipse/Project.java

```diff
@@ -40,13 +57,58 @@ public class Project {
      */
     boolean multiModule;
 
+    /**
+     * Build type associated with this project.
+     */
+    BuildType type;
+
     /**
      * Constructor.
      * 
      * @param project The Eclipse project reference.
      */
     public Project(IProject project) {
         this.iProject = project;
+        type = findBuildType();
```

Comment:
So if a user added pom.xml .. can they refresh our dashboard and have this recalculated?